### ISSUE TITLE
Breaks when the value contains a single quote

### ISSUE THREAD

**Issue by saneef** [charly/nexter#3]
I'm using nexter on a project which needs sort by time (`published_at`) and name of a model - Entry. But, the next breaks when the name contains a single quote.

```
ActiveRecord::StatementInvalid (PG::SyntaxError: ERROR:  syntax error at or near "s"
LINE 1: ...= '2017-04-14 18:06:46 UTC' AND (name > 'children's book Ill...
                                                             ^
```

And the query generated is
```sql
SELECT  "entries".* FROM "entries" WHERE "entries"."feed_id" IN (113629430, 980190962, 298486374, 980190966, 980190972, 980190964) AND (published_at = '2017-04-14 18:06:46 UTC' AND (name > 'children's book Illustrator' OR name IS NULL) OR (published_at < '2017-04-14 18:06:46 UTC')) ORDER BY published_at desc, name asc LIMIT $1)
```

**Comment by charly** [comment]
Hi, I actually had the same problem a few days before you did ! and came up with a quick and dirty fix on the master git branch (see: 4fa5717ed68b4729b4efb63290c4588ce2922686)

Try out `gem 'nexter', git: "https://github.com/charly/nexter"` it should work correctly now, I'll update the gem version soon or try and find a better solution for escaping string for dbs...

**Comment by saneef** [comment]
Thanks! That fix is working. Hope you will be able to find a cleaner fix soon.
